movie_collection.rb

```diff
@@ -51,4 +51,11 @@ def movie_stats(field)
       acc[str_value] += 1
     end
   end
+
+  def accesses_feilds
```

Comment:
1. f`IE`lds not f`EI`lds
2. Consider better naming, it should present all genres